pkg/porter/explain.go

```diff
@@ -169,6 +171,17 @@ func (p *Porter) printBundleExplain(o ExplainOpts, pb *PrintableBundle) error {
 	}
 }
 
+func checkIsInternalParam(inputParam string) bool {
```

Comment:
suggestion: This would read better in English if we changed the name to `isInternalParam`, for example look at line 219 in generatePrintable, it would become `if isInternalParam(p) {`. 

It's a pattern that can be useful at times, naming a function "isNOUN", or "canVERB",  etc.